shared.jq

```diff
@@ -0,0 +1,27 @@
+def apt_get_dist_clean:
+	if [
+		# only suites with APT 2.7.8+ have "apt-get dist-clean"
+		# https://tracker.debian.org/news/1492892/accepted-apt-278-source-into-unstable/
+
+		# https://tracker.debian.org/pkg/apt
+		# https://packages.debian.org/apt
+		"trixie", # TODO once 2.7.8 migrates to testing (and images are rebuilt), this should be removed!
```

Comment:
https://tracker.debian.org/news/1495149/apt-279-migrated-to-testing/ :eyes:

(The next rebuild of Debian can come with a removal of this line :muscle:)